src/ssltunnel.js

```diff
@@ -87,15 +92,32 @@ var TunnelService = {
         return typeof tunneltoken === 'object';
     },
 
-    // method to check if user skipped the sl tunnel setup
+    // method to check if user skipped the ssl tunnel setup
     userSkipped: async function() {
         let notunnel = await Settings.get('notunnel');
         if (typeof notunnel === 'boolean' && notunnel) {
             return true;
         }
 
         return false;
-    }
+    },
+
+    // method to ping the registration server to track active domains
+    pingRegistrationServer: function() {
+        Settings.get('tunneltoken').then((result) => {
```

Comment:
One minor point. Is it necessary to do this I/O every hour and wear out the SD card? :P Can't we store the token in memory and update it when it actually changes (which currently as far as I know it doesn't!).

Reply:
Done.